src/components/SkeletonLoading.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+	<div class="placeholder-main placeholder-main-text">
+		<!-- Placeholder animation -->
+		<template v-for="(suffix, gradientIndex) in ['-regular', '-reverse']">
+			<svg :key="'gradient' + suffix" :class="'placeholder-gradient placeholder-gradient' + suffix">
+				<defs>
+					<linearGradient :id="'placeholder-gradient' + suffix">
+						<stop offset="0%" :stop-color="(gradientIndex === 0) ? colorPlaceholderLight : colorPlaceholderDark" />
+						<stop offset="100%" :stop-color="(gradientIndex === 0) ? colorPlaceholderDark : colorPlaceholderLight" />
+					</linearGradient>
+				</defs>
+			</svg>
+
+			<ul :key="'list' + suffix" :class="'placeholder-list placeholder-list' + suffix">
+				<li v-for="(width, index) in placeholderData" :key="'placeholder' + suffix + index">
```

Comment:
Given that `width` from `placeholderData` is not really used anymore anyway, it should be enough to use `count` for this loop and remove `placeholderData` altogether, no?